firestore/testapp/Assets/Firebase/Sample/Firestore/UIHandler.cs

```diff
@@ -199,12 +199,61 @@ class WaitForTaskCompletion : CustomYieldInstruction {
       }
     }
 
-    protected internal FirebaseFirestore db {
+    protected internal FirebaseFirestore TestFirestore {
```

Comment:
Properties should start with a lowercase letter.

However, I think `db` was ok. I would prefer if we kept it the same.